corehq/apps/case_search/utils.py

```diff
@@ -212,11 +216,11 @@ def _get_query(self, criteria):
             return self._get_daterange_query(criteria)
 
         value = self._remove_ignored_patterns(criteria.key, criteria.value)
-        fuzzy = criteria.key in self._fuzzy_properties
         if criteria.is_ancestor_query:
             query = f'{criteria.key} = "{value}"'
-            return build_filter_from_xpath(self.query_domains, query, fuzzy=fuzzy)
+            return build_filter_from_xpath(self.query_domains, query, self.fuzzy_properties)
```

Comment:
I'm not totally sure I get this. 
If the ancestor is a different case type, shouldn't you be looking at a different set of fuzzy properties?

Reply:
hmm, great point - I'll have to think about that a bit

Reply:
for now fuzzy matching won't work for ancestor queries 8d5f63dbbf2